internal/services/springcloud/spring_cloud_gateway_resource.go

```diff
@@ -179,16 +179,10 @@ func resourceSpringCloudGateway() *pluginsdk.Resource {
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
 						"cpu": {
-							Type:     pluginsdk.TypeString,
-							Optional: true,
-							Default:  "1",
-							ValidateFunc: validation.StringInSlice([]string{
-								"500m",
-								"1",
-								"2",
-								"3",
-								"4",
-							}, false),
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							Default:      "1",
+							ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
we should add a comment here explaining that additional values _may_ be possible via the Service Team - so we're intentionally not validating this